cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -193,8 +194,12 @@ async function begin() {
   const requestedTestIds = argv._;
   const {default: rawTestDefns} = await import(url.pathToFileURL(testDefnPath).href);
   const allTestDefns = updateTestDefnFormat(rawTestDefns);
+  const excludedTests = new Set(exclusions[argv.runner] || []);
+  const filteredTestDefns = !excludedTests.size ? allTestDefns :
+    allTestDefns.filter(test => !excludedTests.has(test.id));
```

Comment:
I think you're actually going to have to pass this into `getDefinitionsToRun` since if `invertMatch` is true this is actually going to add the excluded tests to the list of tests to run

Reply:
In https://github.com/GoogleChrome/lighthouse/issues/14127#issuecomment-1159108240 I suggested adding it to just this branch in `getDefinitionsToRun`

https://github.com/GoogleChrome/lighthouse/blob/ec4b36b8dcbcf6c68a20df5cc79b3acfde9f7760/cli/test/smokehouse/frontends/smokehouse-bin.js#L52-L55

still seems reasonable so that you could still run `yarn smoke --runner devtools redirects-scripts` manually if the need ever arose.

If we wanted to support exclusions with `invertMatch` _and_ the explicit case that gets a lot tricker, so maybe just punt on exclusions+`invertMatch` for when/if we actually need that?

Reply:
What would be a good use case for `invertMatch` now? Previously it was being used to exclude tests.

Reply:
@brendankenny `getDefinitionsToRun` runs off the supplied list, which is already filtered. So the scenario of inversion adding excluded list wouldn't happen. The other case though, which is wanting to run an excluded test manually, wouldn't work, and will require removal of exclusion from config.

Reply:
> `getDefinitionsToRun` runs off the supplied list, which is already filtered

whoops, yes, you're totally right

> The other case though, which is wanting to run an excluded test manually wouldn't work, and will require removal of exclusion from config.

I guess I thought we were doing this from https://github.com/GoogleChrome/lighthouse/issues/14127#issuecomment-1159108240, instead of having an explicit `--ignore-exclusions` flag. If someone is e.g. trying to get `byte-efficiency` working in devtools, running `yarn smoke --runner devtools byte-efficiency` simply won't run anything. It's not the end of the world to remember to go remove it from the exclusions list at that point, and I guess we can also just see how it goes in practice, but it also seems like it wouldn't be too hard to support?

Reply:
> What would be a good use case for `invertMatch` now? Previously it was being used to exclude tests.

Probably not much. It was originally added for a weaker and much more awkward version of what sharding does now. We can survey if anyone else wants it (and double check if LR is using it/needs it for now), but otherwise removing SG